configs/default/IFRC-IFLIGHT_F745_AIO.config

```diff
@@ -108,26 +118,40 @@ dma pin A03 0
 # feature
 feature -RX_PARALLEL_PWM
 feature RX_SERIAL
+feature TELEMETRY
 feature LED_STRIP
 feature OSD
+feature ESC_SENSOR
 
 # serial
-serial 2 64 115200 57600 0 115200
+serial 0 8192 115200 57600 0 115200
+serial 1 64 115200 57600 0 115200
+serial 5 1024 115200 57600 0 115200
+serial 6 2 115200 57600 0 115200
 
 # led
 led 0 6,11::CTOBVIW:0
 led 1 7,11::CTOBVIW:0
 led 2 8,11::CTOBVIW:0
 led 3 9,11::CTOBVIW:0
 
+# aux
+aux 0 0 0 900 1300 0 0
```

Comment:
Seems like a very bad idea to pre-configure an arming switch. You have no way of knowing the user's radio setup and aux1 might be used for something else causing an unsafe situation.

Reply:
Thanks. Removed it now.